CHANGES.rst

```diff
@@ -124,6 +127,9 @@ astropy.table
   index is now handled in the conversion methods. [#8247]
 - Added ``rename_columns`` method to rename multiple columns in one call. [#5159, #8070]
 
+- ``astropy.constants``
+
+- Allow to pass ``htmldict`` option to the jsviewer writer. [#6551]
```

Comment:
@Cadair - looks like there was some kind of issue here with the rebase?

Reply:
:roll_eyes: stupid changelog